README.md

```diff
@@ -4,63 +4,65 @@
 const topic = 'dom-events';
-mailbox.subscribe(topic, function(_, data) {
+mailbox.subscribe(topic, (_, data) => {
   console.log(data);
 });
 ```
 
 
-## Use with ZAP
-
-#### Inject in all responses
+#### Import in your application
 
-[inject.js](https://github.com/zaproxy/community-scripts/blob/master/httpsender/inject_js_in_html_page.js) is an HTTP Sender script (see more [here](https://github.com/zaproxy/community-scripts/tree/master/httpsender)) to inject any kind of Javascript content into webpages that interest us.
-
-Requirements:
-
-  * Install the `Script Console` from the add-on marketplace.
-  * Install the `Community Scripts` from the add-on marketplace.
+##### If you use a Javascript bundler
 
-Replace `/tmp/test.js` by the **absolute path** of the generated `bundle.js` from this project.
+Such as [webpack](https://webpack.github.io/), [rollup](https://rollupjs.org/guide/en), [browserify](http://browserify.org/), ...
+All you need to do is to require this package in you application.
 
-
-## Build the app (standalone)
-
-```
-npm install
-npm run build
+```bash
+require('@zaproxy/front-end-tracker');
 ```
 
-  * [Browserify](https://github.com/browserify/browserify) takes care of generating the bundle to be injected into webpages.
-  * [uglify-es](https://www.npmjs.com/package/uglify-es) takes care of minifying the bundle to be injected into webpages.
 
-Then `dist/front-end-tracker.js` contains the application to be imported into the webpages you want to track.
+##### If you do not
 
+There is a [pending feature request](https://github.com/zaproxy/front-end-tracker/issues/5) to make a bundled version of this package available from a CDN.
 
-## Dev
 
-```
-npm install
-npm run watch
-```
+#### Import in any HTML page
 
-[Watchify](https://github.com/browserify/watchify) takes care of the rebuild on file changes and generates the `bundle.js` to be injected into webpages.
+You can use ZAP to inject a bundle of this package in HTTP responses.
 
+[inject.js](https://github.com/zaproxy/community-scripts/blob/master/httpsender/inject_js_in_html_page.js) is an HTTP Sender script (see more [here](https://github.com/zaproxy/community-scripts/tree/master/httpsender)) to inject any kind of Javascript content into webpages that interest us.
 
-## Resources
-
-  * [Full list of DOM Events](https://developer.mozilla.org/en-US/docs/Web/Events)
-  * [webcompat/tinker-tester-developer-spy](https://github.com/webcompat/tinker-tester-developer-spy): some hooks have been copy/pasted in the [hooks folder](https://github.com/zaproxy/front-end-tracker/tree/master/hooks).
+  * Install the `Script Console` from the add-on marketplace.
+  * Install the `Community Scripts` from the add-on marketplace.
+  * Bundle the tracker from the source
+  ```bash
+  git clone git@github.com:zaproxy/front-end-tracker.git
+  cd front-end-tracker
+  npm install
+  npm run build
+  ```
+  * Copy the bundled content to the imported location: `cp bundle.js /tmp/test.js`
```

Comment:
Did you mean `dist/front-end-tracker.js` instead of `bundle.js`? The latter does not exist after running the above commands.

Reply:
Right.